File: app/src/main/java/com/dranilsaarias/nad/MyApp.java
package com.dranilsaarias.nad;

import android.app.Application;

/**
 * Created by pico on 9/9/17.
 */

public class MyApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        TypefaceUtil.overrideFont(getApplicationContext(), "SERIF", "fonts/futurabt_book.otf"); // font from assets: "assets/fonts/Roboto-Regular.ttf
    }
}

File: app/src/main/java/com/dranilsaarias/nad/MyApp.kt
package com.dranilsaarias.nad

import android.app.Application


class MyApp : Application() {

    override fun onCreate() {
        super.onCreate()
        TypefaceUtil.overrideFont(applicationContext, "SERIF", "fonts/futurabt_book.otf") // font from assets: "assets/fonts/Roboto-Regular.ttf
    }
}
